src/App.jsx

```diff
@@ -25,7 +25,8 @@ function KanbanBoard() {
     deleteTask,
     addTask,
     addLabel,
-    removeLabel
+    removeLabel,
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
try {
  await removeLabel(selectedTask.columnId, taskId, labelId);
} catch (error) {
  console.error('Failed to remove label:', error);
  // Handle error appropriately
}
```
Consider adding error handling for the removeLabel function calls to handle potential failures gracefully and provide user feedback.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;